src/metabase/api/activity_log.clj

```diff
@@ -0,0 +1,275 @@
+(ns metabase.api.activity-log
```

Comment:
Are this namespace, the model namespace, and the tests all just copied over from the `activity` namespaces verbatim? Or are there changes?

If it's the former, could you just rename the `activity` namespaces in place, and just modify the code directly to work with the new table?

Reply:
Yea, that might be a good idea. I can try that out soon.